KILTprotocol/sdk-js
# Mn 176 - doc public methods AttestedClaim

## fixes #176  

## How to test:  
If you'd like to see how the doc looks like:  
Run `npm run build:docs`. This generates the doc at `sdk-js/docs/api`.
Open the so-generated files in your browser: `sdk-js/docs/api/classes/attestedclaim.attestedclaim-1.html`.

## Checklist:

- [x] I have verified that the code works // tests ran
